server/app/views/applicant/ApplicantProgramSummaryView.java

```diff
@@ -118,10 +113,17 @@ public Content render(Params params) {
 
     params.bannerMessage().ifPresent(bundle::addToastMessages);
 
+    String pageTitle =
+        params.inReview()
+            ? messages.at(MessageKey.TITLE_PROGRAM_REVIEW.getKeyName())
+            : messages.at(MessageKey.TITLE_PROGRAM_PREVIEW.getKeyName());
+    bundle.setTitle(
+        layout.renderPageTitleWithBlockProgress(
+            pageTitle, params.completedBlockCount(), params.totalBlockCount(), true));
```

Comment:
That was purely an accident, thanks for pointing it out :D.

Fixed!

Reply:
Isn't this now: "Program application review - My test program - 3 of 5" where 3 is the number of blocks they've completed? I'm worried the "3 of 5" might be confusing on the review page, since it's not super clear what it's indicating. Imo, maybe reverting it to "Program application review - My test program" would be clearer.

Reply:
I had totally misunderstood. Yah, there's no reason to add the block count to the title for preview / review since we didn't do it before.

I did the following and recorded the titles before:
  * Navigated to the preview page (recorded title)
    - Program application preview — Mock program — CiviForm
  * Filled out the first block of information and clicked "Next"
  * Arrived at the second block page (recorded title)
    - Mock program — 2 of 5 — CiviForm
  * Clicked the review button (recorded title)
    - Program application review — Mock program — CiviForm
  * Filled out the rest of the blocks and arrived at the review page (recorded title)
    - Program application review — Mock program — CiviForm

I've reverted so that the only caller of `renderPageTitleWithBlockProgress` is the block edit view. I did keep the change in to move the method to the base class so that the logic for determining the block count could be alongside where we render it in the UI.

Reply:
Cool, this looks great!